app/controlplane/api/workflowcontract/v1/crafting_schema.proto

```diff
@@ -103,3 +105,32 @@ message Annotation {
   // This value can be set in the contract or provided during the attestation
   string value = 2;
 }
+
+// A policy to be applied to this contract
+message PolicyAttachment {
+  // policy reference. It must consist of an identifier or URI reference
+  string ref = 1 [(buf.validate.field).string.min_len = 1];
+
+  // rules to select a material or materials to be validated by the policy.
+  // If none provided, the whole statement will be injected to the policy
+  MaterialSelector selector = 2;
+
+  // set to true to disable this rule
+  bool disabled = 3;
+
+  repeated PolicyArgument with = 4;
+
+  message MaterialSelector {
+    // material name
+    string name = 1;
+
+    // material type
+    CraftingSchema.Material.MaterialType kind = 2;
```

Comment:
does this even make sense? I thought that policies by itself already define the material kind?

Reply:
Yes, maybe we don't need it yet. I was thinking of the case of "apply this policy to all materials of this type". The policy itself could be something generic, like "check that the format is json". But we might be making it too complex. I'll remove this.